README.md

```diff
@@ -29,6 +29,21 @@ To run the project locally at `http://localhost:3000`
 bun run dev -o
 ```
 
+## Troubleshooting Guide
+
+If you encounter issues during the installation, refer to the troubleshooting steps in the table below.
+
+| Issue | Possible Cause | Solution |
+|-------|----------------|----------|
+| **Network Timeout** | Slow or unstable internet connection | Increase the timeout duration by using: `curl -fsSL --connect-timeout 60 https://bun.sh/install | bash` |
+| **Manual Download Required** | Automatic download fails | Manually download the Bun binary from the [Bun releases page on GitHub](https://github.com/oven-sh/bun/releases/latest). Extract the binary and place it in a directory that is in your system's PATH. |
+| **Permissions Issue** | Bun binary is not executable | Ensure the Bun binary is executable by running: `chmod +x ~/.bun/bin/bun` |
+| **PATH Not Updated** | `~/.bun/bin` not in PATH | Verify and manually add it to your PATH by editing your shell configuration file (`~/.zshrc` for zsh): `echo 'export PATH="$HOME/.bun/bin:$PATH"' >> ~/.zshrc && source ~/.zshrc` |
+| **TTY Initialization Failed** | Terminal handling issues or TTY not supported | Run without TTY by setting the environment variable: `NO_TTY=1 bun install` or disable telemetry: `export NUXT_TELEMETRY_DISABLED=1` |
+| **Telemetry Prompt Issue** | Nuxt telemetry prompt causing issues | Set an environment variable to disable the telemetry prompt: `export NUXT_TELEMETRY_DISABLED=1` |
+| **Conflicting Aliases** | Conflicts with existing aliases or functions | Ensure there are no conflicting aliases or functions named `bun` by running: `type bun` |
+| **Reinitialize Shell** | Shell environment not refreshed | Completely reinitialize your shell environment by starting a new shell session: `exec /bin/zsh` |
+
```

Comment:
A Troubleshooting Guide should me more specific to our project and setup. This block of content is specifically for bun and only for if you installed it via curl. There are five different options for installing bun on their website. 
We should not create a troubleshooting guide for debugging a third party tool in our projects. This provides a brittle dependency that requires us to maintain documentation for a tool we do not own and require checking to make sure it does not become stale or out of sync with the official documentation.

Reply:
Okay. 

I created the troubleshooting guide to assist other users who might encounter similar issues running Bun on their PC since the project primarily uses Bun.

Additionally, there will be a troubleshooting guide in the documentation proper for zkSync

Reply:
Yes, our project does use Bun, but we can't maintain troubleshooting the use of the tool itself. We do not maintain Bun and we cannot cover all potential use cases of setting up Bun in our own docs. We need to leave that to the Bun community and the user.

Reply:
Ok